sync/modules/user-policy.hcl.j2

```diff
@@ -22,10 +22,12 @@ path "sys/mounts/*"
 # need to use `data/` because we're on secrets engine v2
 # `data/` will not appear in the actual usage of vault since most clients
 # intelligently add it in. it only appears in the policy
-path "secrets/data/{{ team_name }}-*"
+{% for repo in repos %}
+path "secrets/data/{{ repo }}"
```

Comment:
do `{{ repo }}-*` so things like PCX can have multiple secrets

Reply:
Wouldn't that prevent them from using a secret called `penn-courses`? They'd have to add some suffix, right?

Reply:
If you look at policy rn, I have 2 rules. one for team name and one for `teamname-*`. I'd recommend doing the same here just with product names.

Reply:
Updated